aiohttp_session/__init__.py

```diff
@@ -36,6 +44,14 @@ def new(self):
     def identity(self):
         return self._identity
 
+    @property
+    def created(self):
+        return self._created
+
+    @property
+    def is_empty(self):
```

Comment:
Ok than.
But name should be changed to `.empty` -- like `.changed` and `.created`